web_src/js/features/tableSort.js

```diff
@@ -0,0 +1,68 @@
+export default function initTableSort() {
+  $('th[data-sortt-asc]').each(function () {
+    // get data
+    const { sorttAsc, sorttDesc, sorttDefault } = $(this)[0].dataset;
+
+    // add onclick event
+    $(this).on('click', () => {
+      tableSort(sorttAsc, sorttDesc, sorttDefault);
+    });
+
+    // add arrow to column
+    const arrow = getArrow(sorttAsc, sorttDesc, sorttDefault);
+    // if function got a match ...
+    if (arrow !== false) {
+      $(this).prepend(`${arrow} `);
+    }
+  });
+}
+
+function tableSort(normsort, revsort, isdefault) {
+  if (!(normsort)) return false;
+  if (!(revsort)) revsort = '';
+
+  const url = new URL(window.location);
+  let url_sort = url.searchParams.get('sort');
+  if ((url_sort === null) && isdefault) url_sort = normsort;
+
+
+  if (url_sort !== normsort) {
+    url.searchParams.set('sort', normsort);
+  } else if (revsort !== '') {
+    url.searchParams.set('sort', revsort);
+  }
+
+  window.location.replace(url.href);
```

Comment:
```suggestion
  window.history.replaceState(null, null, url.href);
```

Reply:
this doesn't reload the page witch is needed to sort in a different way

-> reverted 98836407e6a6f6687feaf5fd7ae2392cba833131

Reply:
Oh, I thought it was actually doing client-side sorting, my bad.